static-site/src/sections/edit-group-settings-page.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import * as splashStyles from "../components/splash/styles";
+import { ErrorBanner } from "../components/splash/errorMessages";
+import GenericPage from "../layouts/generic-page";
+import { FlexGridRight, HugeSpacer, MediumSpacer } from "../layouts/generalComponents";
+
+const UNAUTHORIZED_MESSAGE = `
+  You must have the owners role within a group to edit group settings.
+  If your permissions have changed recently, try logging out and logging back in.
+`;
+
+const EditGroupSettingsPage = ({ location, groupName }) => {
+  const [ userAuthorized, setUserAuthorized ] = useState(true);
+
+  useEffect(() => {
+    const checkUserAuthz = async () => {
+      if (! await canUserEditGroupSettings(groupName) && !cleanUp){
+        setUserAuthorized(false);
+      }
+    }
+    let cleanUp = false;
+    checkUserAuthz();
+    return () => cleanUp = true;
+  }, [groupName]);
+
+  return (
+    <GenericPage location={location}>
+      <FlexGridRight>
+        <splashStyles.Button to={`/groups/${groupName}`}>
+          Return to "{groupName}" Page
+        </splashStyles.Button>
+      </FlexGridRight>
+      <MediumSpacer/>
+
+      <splashStyles.H2>
+        Editing "{groupName}" Group Settings
+      </splashStyles.H2>
+      <HugeSpacer/>
+
+      {userAuthorized
+        ? <p>Placeholder for editing forms.</p>
+        : <ErrorBanner title={UNAUTHORIZED_MESSAGE}/>}
+    </GenericPage>
+  )
+};
+
+export const canUserEditGroupSettings = async (groupName) => {
+  try {
+    const groupOverviewOptions = await fetch(`/groups/${groupName}/settings/overview`, { method: "OPTIONS" });
+    const allowedMethods = groupOverviewOptions.headers.get("Allow");
+    if (allowedMethods) {
+      const editMethods = ["PUT", "DELETE"];
+      return editMethods.every((method) => allowedMethods.includes(method));
+    }
+    return false;
```

Comment:
Ah, yup! `String.includes()` might have "false" matches because the searched string can be anywhere in the given string. It's better to be more precise and split the string before inclusion checks.

Reply:
Yep. I mean, pretty unlikely we'd actually hit unintended matches, but good to DTRT and close off a whole class of bug from the start.

Reply:
Updated in 55a7d4a8868e215a06fd56f4528dd00c556b4fd0